ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
> @seandst grabbing that from a config file will even help testing if the Pulp Server is reporting the right version. This was a design decision within Pulp Smash I'm just replicating it to the play book that runs it.

As the system under test, I don't trust Pulp. I don't even trust it to correctly state its own version, nor do I trust it to be packaged correctly. There may also be times when we want to run the test suite against a different version of Pulp - say, to see how much difference there is in compatibility between two versions of Pulp.